diff --git a/sdk/objc/components/audio/RTCAudioSessionConfiguration.m b/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
index 74de6b5..143ce29 100644
--- a/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
+++ b/sdk/objc/components/audio/RTCAudioSessionConfiguration.m
@@ -62,3 +62,7 @@ - (instancetype)init {
     _category = AVAudioSessionCategoryPlayAndRecord;
+#if TARGET_OS_TV
+    _categoryOptions = 0;
+#else
     _categoryOptions = AVAudioSessionCategoryOptionAllowBluetooth;
+#endif
 
